editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -1205,25 +1221,53 @@ function createExpressionOtherJavaScript(
   sourceMap: RawSourceMap | null,
   elementsWithin: ElementsWithin,
   alreadyExistingUIDs: Set<string>,
-): WithParserMetadata<JSExpressionOtherJavaScript> {
+): WithParserMetadata<JSExpressionMapOrOtherJavascript> {
+  const isList = (() => {
+    if (!TS.isCallExpression(node) || !TS.isPropertyAccessExpression(node.expression)) {
+      return false
+    }
+    const funcNode = node.expression.getChildAt(2, sourceFile)
+    return funcNode?.getText(sourceFile) === 'map'
+  })()
+
   // Ideally the value we hash is stable regardless of location, so exclude the SourceMap value from here and provide an empty UID.
-  const value = jsExpressionOtherJavaScript(
-    javascript,
-    transpiledJavascript,
-    definedElsewhere,
-    null,
-    elementsWithin,
-    '',
-  )
+  const value = isList
```

Comment:
Huh, I'm not sure why both `value` and `expression` are almost identical here. It looks like that was maybe the result of refactoring over time (possibly from when the 2 arbitrary JS types were merged)